.github/workflows/kani.yml

```diff
@@ -35,4 +35,11 @@ jobs:
           submodules: true
 
       - name: Run Kani Script
-        run: bash ./head/scripts/check_kani.sh ${{github.workspace}}/head
+        run: bash ./head/scripts/run-kani.sh --path ${{github.workspace}}/head
+
+      - name: Run Kani Script with --kani-args and -p
+        run: bash ./head/scripts/run-kani.sh -p ${{github.workspace}}/head --kani-args --harness ptr::verify::check_read_u128 --harness ptr::unique::verify::check_as_mut
+
+      - name: Run Kani Script without -p
+        working-directory: ${{github.workspace}}/head
```

Comment:
Couldn't the testing of the script itself be done in a separate job?

Reply:
I'd initially put it in a separate job, but merged it all into one based on this comment https://github.com/model-checking/verify-rust-std/pull/78#discussion_r1747802242

Reply:
Hmm, but given it's testing the script itself, shouldn't we have a workflow that doest just the script testing, and that workflow should only execute when any of the files relating to the script (the script itself plus toml files perhaps) change?

Reply:
Just to clarify, I wasn't saying they had to be merged in the same job. It just wasn't clear to me what each workflow was doing.

I would suggest that you test the script as a separate job of this workflow. But make it clear what each step is testing, so people are not confused. Maybe add some comments explaining what this is testing.